dbt_platform_helper/utils/validation.py

```diff
@@ -677,13 +680,35 @@ def _validate_environment_pipelines_triggers(config):
         abort_with_error(error_message + "\n  ".join(errors))
 
 
+def lint_yaml_for_duplicate_keys(file_path):
+
+    lint_yaml_config = """
+rules:
+  key-duplicates: enable
+"""
+    yaml_config = config.YamlLintConfig(lint_yaml_config)
+
+    with open(file_path, "r") as yaml_file:
+        file_contents = yaml_file.read()
+        results = linter.run(file_contents, yaml_config)
+
+    parsed_results = ["\t" + f"Line {result.line}: {result.message}" for result in results]
```

Comment:
Can also remove the ` in mapping (key-duplicates)` to make the message clearer

Reply:
Hopefully your tests would catch any kind of change like that. It would need a version bump of yamllint so you should know.